agents/builtin/mongodb/internal/profiler/profiler_test.go

```diff
@@ -15,3 +15,74 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 package profiler
+
+import (
+	"testing"
+	"time"
+
+	"github.com/percona/pmgo"
+	"github.com/percona/pmm/api/qanpb"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/percona/pmm-agent/agents/builtin/mongodb/internal/profiler/aggregator"
+	"github.com/percona/pmm-agent/agents/builtin/mongodb/internal/report"
+)
+
+func TestProfiler(t *testing.T) {
+	defer func() {
+		aggregator.DefaultInterval = time.Duration(time.Minute)
+	}()
+	aggregator.DefaultInterval = time.Duration(time.Second)
```

Comment:
It is better to do something like:
```go
defaultInterval := aggregator.DefaultInterval
aggregator.DefaultInterval = time.Duration(time.Second)
defer func() { aggregator.DefaultInterval = defaultInterval }
```

That way we restore the previous value, not the value specified in the other package which should be synced on change.

Reply:
Fixed. In the old code, it was a part of the configuration in constructors and it was more clear. But then I have removed not used parameters and only 2 of them remained. So I move it to package level constant and then in package level variable to have a possibility to change in in tests.